packages/manager/apps/container/src/container/legacy/server-sidebar/order/shop-config/order.constants.ts

```diff
@@ -386,19 +384,19 @@ export const ORDER_URLS: Record<
     },
     dedicated_cloud: {
       ASIA:
-        'https://www.ovhcloud.com/en-sg/enterprise/products/hosted-private-cloud/prices/',
+        'https://www.ovhcloud.com/asia/hosted-private-cloud/vmware/prices/',
```

Comment:
Can you please add India specific link ?

```suggestion
        'https://www.ovhcloud.com/asia/hosted-private-cloud/vmware/prices/',
```